docs/tutorials/hostpath/nodeaffinitylabels.md

```diff
@@ -0,0 +1,25 @@
+# Use Node-Selector labels
+
+Hostpath LocalPV uses the Kubernetes Node label(s) (For example: `kubernetes.io/hostname=<node-name>`) to uniquely identifly a node.
+
+In some cases, this label (`hostname`) is not unique across all the nodes in the cluster. This was seen on clusters provisioned with [Bosh](https://bosh.io/docs/) across different fault domains.
+
+A unique Node label or set of labels may be used instead of the above mentioned Kubernetes default label to uniquely identify a node. This label(s) may be set by you, the administrator.
```

Comment:
A unique Node label or set of labels may --> A unique Node label (or set of labels) may